gaia-dialog-alert.js

```diff
@@ -17,6 +17,8 @@ module.exports = component.register('gaia-dialog-alert', {
     this.els = {
       dialog: this.shadowRoot.querySelector('gaia-dialog')
     };
+    this.els.dialog.addEventListener('closed',
+                                     GaiaDialogProto.hide.bind(this));
```

Comment:
unless app code calls it we never reach line 31.
The onclick event handler is in the "nested" dialog.